app/views/layouts/design_system.html.erb

```diff
@@ -1,5 +1,11 @@
 <% environment = GovukPublishingComponents::AppHelpers::Environment.current_acceptance_environment %>
 
+<% if user_signed_in? && current_user.has_email? %>
```

Comment:
What's the significance of using `current_user.has_email?` here?

Is there a case where a user will be signed in _without_ an email address? And if so, why don't we track their organisation slug (if they have one)?

Reply:
Also, it'd be nice to adapt/reuse/duplicate [this integration test](https://github.com/alphagov/whitehall/blob/c0ad13dd04e6c2f3b53cd4373379a0b0f3dc307f/test/integration/admin_analytics_test.rb) which covers the Bootstrap version of this behaviour.

Reply:
From what I can tell there may be a case where a signed in user has no email address, but I don't see a reason not to track their organisation slug in this case. This logic was used in the old admin layout file, they don't give any indication why it's in there in the PR and I've checked the integration tests and they don't test for a case where the user has no email, so I'm assuming it's safe to remove. Thanks for flagging @ollietreend